src/components/app.component.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import * as S from './styled.component';
+import Snake from './snake/Snake.component';
+import GameOver from './gameOverWindow/GameOver.component';
+import Settings from './settings/Settings.component';
+import {
+  parameters,
+  snakeColor,
+  saveItem,
+  settings
+} from '../constants/constants';
+import { spawnFood } from '../utilities/spawnFood';
+import { createSnake } from '../utilities/createSnake';
```

Comment:
In this case, when you have a lot of utility functions, you can write an index file and export those utilities.

`import { spawnFood, createSnake, ... } = '../utils';`